wiki/Tournaments/TSC/1/en.md

```diff
@@ -0,0 +1,99 @@
+---
+tags:
+- Taiko Suiji Cup
+- TSC
+---
+# Taiko Suiji Cup
```

Comment:
I'm unsure, because `Taiko Suiji Cup` is the original name.
But i can't have "Yearly folder naming", so i use numbering folder instead (like i did on Shift Cup a while ago)